solaris/tile/raster_tile.py

```diff
@@ -45,17 +45,17 @@ class RasterTiler(object):
     force_load_cog : bool, optional
         If `src` is a cloud-optimized geotiff, use this argument to force
         loading in the entire image at once.
-    aoi_bounds : list, optional
+    aoi_boundary : list, optional
```

Comment:
I think this needs a bit more since it's no longer limited to just a list-like with four coordinates, right? Can be a polygon, etc.?

Reply:
Ah missed that one, thanks for the catch.